### PR TITLE
Fix all the grammar problem,improve text and more.

### PR DESCRIPTION
**Pull request by SistineFibelKen**
According to the #488 request I fix all the grammar problem and improve text and more. If you from Mainland China and want to improve the translations please contact me frist. thanks.

E-mail：yu2221038595@163.com

### CODE REVIEW

**Comment by SistineFibelKen** [comment]
@Alexander01998 😋 Don't forget it ~

**Review by Alexander01998 — CHANGES_REQUESTED** [review]
😅 Here you go. @SistineFibelKen

File: src/main/resources/assets/wurst/lang/zh_cn.json [hunk 1]
```diff
@@ -1,139 +1,139 @@
...
+  "description.wurst.hack.autoswim": "当你在水中会自动切换泳姿状态。",
+  "description.wurst.hack.autoswitch": "自动更换手上拿着的物品。\n\n§l进阶提示：§r 倘若在你的快捷栏有不同颜色的羊毛以及，不同颜色的混粘土，利用这个功能与 BuildRandom 同时使用会出现意想不到的效果。",
+  "description.wurst.hack.autosword": "当你击杀实体的时候，自动使用快捷栏中最好的武器。\n提示： 在使用 Killaura 功能时候，最好开启。",
+  "description.wurst.hack.autotool": "当你破坏方块的时候会在你的物品快捷栏中切换成最优的破坏工具。",
+  "description.wurst.hack.autototem": "当你即将死亡的时候自动将图腾移动到副手上。",
+  "description.wurst.hack.autowalk": "让你自动走路，不需要自己按着W键。",
+  "description.wurst.hack.basefinder": "通过寻找人为建造的方块以找到玩家的基地\n假如方块被找到将会高亮以你所选的颜色。\n建议用于寻找帮派的基地。",
+  "description.wurst.hack.blink": "暂停所有动作更新,并在关闭时继续。",
+  "description.wurst.hack.boatfly": "允许玩家在使用船飞天。",
+  "description.wurst.hack.bonemealaura": "在指定的作物上自动使用骨粉。\n使用 Checkboxes 来指定作物。",
```
[the remark this answers is not in the document]
Comment (SistineFibelKen) [reply src/main/resources/assets/wurst/lang/zh_cn.json:34]: is it the Checkboxes likes Minecraft Debug tools?

File: src/main/resources/assets/wurst/lang/zh_cn.json [hunk 2]
```diff
@@ -1,139 +1,139 @@
...
+  "description.wurst.hack.bowaimbot": "当你用弓或者弩时自动瞄准所设定的实体。",
+  "description.wurst.hack.buildrandom": "在你的附近随机放方块。",
+  "description.wurst.hack.bunnyhop": "让你自动跳跃，但不能做到像CSGO一样。",
+  "description.wurst.hack.cameranoclip": "允许第三人称忽视墙体碰撞直接把视角穿过去。",
+  "description.wurst.hack.cavefinder": "帮助你找到洞穴并以你所选的颜色将他们高亮。",
+  "description.wurst.hack.chattranslator": "从聊天栏的信息利用Google翻译成你所指定的语言(在中国大陆内不起效,除非你懂得如何重新起效)。",
+  "description.wurst.hack.chestesp": "按你所设定的颜色高亮附近的箱子,潜影盒,陷阱箱,末影箱。",
+  "description.wurst.hack.clickaura": "当瞄准的时候。自动攻击你附近有效的实体\n\n§c§l警告：§r 此功能通常来说比 Killaura 更加容易被发现且被检测出来，这极力推荐使用 Killaura 或者 TriggerBot 作为替代。",
+  "description.wurst.hack.clickgui": "基于类Windows样点击版GUI(有分类)。",
```
[the remark this answers is not in the document]
Comment (SistineFibelKen) [reply src/main/resources/assets/wurst/lang/zh_cn.json:43]: sry about that , I always thought that was like Windows™ operating system 🤣  (win10 of windows) , Turn out it was like window GUI.

File: src/main/resources/assets/wurst/lang/zh_cn.json [hunk 3]
```diff
@@ -1,139 +1,139 @@
...
+  "description.wurst.hack.itemgenerator": "生成随机的物品并将其丢在地上。\n§o仅限创造模式。§r",
+  "description.wurst.hack.jesus": "允许你走在水面上。\n耶稣曾经用这个功能 约~2000 年前。",
+  "description.wurst.hack.jetpack": "模仿你好像真的有飞行背包一样飞行。\n\n§c§l警告：§r 如果你不使用 NoFall，会导致受到掉落伤害。",
+  "description.wurst.hack.kaboom": "像爆炸一样破坏你附近的方块。\n这个可能比 Nuker 功能破坏更快如果服务器没有反作弊的话，使用适合的工具和破坏脆弱的方块可以事半功倍。\n注意：这并不是真正的爆炸。",
+  "description.wurst.hack.killauralegit": "Killauralegit 更难被反作弊检测。\n在正常的NoCheat+(NCP)服务器上不需要！",
+  "description.wurst.hack.killaura": "自动攻击你附近的实体。",
+  "description.wurst.hack.killpotion": "生成一个能杀死绝大多数生物的药水，包括创造模式的玩家，但对那些不死的生物或者已死生物无效。\n\n需要在创造模式下。",
+  "description.wurst.hack.liquids": "取消限制，允许你在水中直接放置方块。",
+  "description.wurst.hack.lsd": "产生吃了云南的蘑菇一样的效果。",
+  "description.wurst.hack.masstpa": "给每个人发送tpa请求。\n直到有人接受就会停止\n\n§c§l警告：§r国内服务器使用会因为Wurst判断的是英文而导致tpa仍在继续。",
```
[the remark this answers is not in the document]
Comment (SistineFibelKen) [reply src/main/resources/assets/wurst/lang/zh_cn.json:81]: I will check this more details next weekend.

File: src/main/resources/assets/wurst/lang/zh_cn.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (Alexander01998) [reply src/main/resources/assets/wurst/lang/zh_cn.json]: It's just a box like this:

- [X] checked
- [ ] not checked

File: src/main/resources/assets/wurst/lang/zh_cn.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (SistineFibelKen) [reply src/main/resources/assets/wurst/lang/zh_cn.json]: Forget to fix this clickgui description 🤣

**Review by Alexander01998 — APPROVED** [review]
All fixed as far as I can tell.

**Comment by SistineFibelKen** [comment]
> All fixed as far as I can tell.

thanks for that , maybe it was the finally version , should I leave e-mail address in the text ?

**Comment by Alexander01998** [comment]
@FeyXieXzf @SiongSng how is the Chinese grammar now?

**Comment by Alexander01998** [comment]
@Aikini you did the Chinese translations for Mo Glass. I thought you might want to join this conversation as well.

**Comment by Aikini** [comment]
> @Aikini you did the Chinese translations for Mo Glass. I thought you might want to join this conversation as well.

I don't have enough time to examine them lately, but they may looks fine in grammar.

**Comment by FeyXieXzf** [comment]
I think there are still some translations need to be fixed. Also, you need to add spaces between Chinese words and English words, like the description in AntiBlind.

**Comment by FeyXieXzf** [comment]
And still some weird grammar mistake.

**Comment by Alexander01998** [comment]
> And still some weird grammar mistake.

Can you please give specific examples of this? Maybe post a review with comments on the affected lines.

**Comment by FeyXieXzf** [comment]
For the space mistake, line 4.
For the weird words, line 34, it says "Checker" instead of "Checkboxes"

**Comment by FeyXieXzf** [comment]
Also, in line 53, it should just say "Nuker" instead of "the Nuker function".

**Comment by Alexander01998** [comment]
> For the space mistake, line 4.

Fixed.

> For the weird words, line 34, it says "Checker" instead of "Checkboxes"
> Also, in line 53, it should just say "Nuker" instead of "the Nuker function".

@SistineFibelKen

**Comment by SistineFibelKen** [comment]
> Also, in line 53, it should just say "Nuker" instead of "the Nuker function".

`允许你更快的破坏方块。\n提示：可以和 Nuker 功能配合利用。` 

` Nuker 功能。` is mean that "Nuker function"

Are u sure u r come from China...?🤔(just my personal question) @FeyXieXzf 

> For the weird words, line 34, it says "Checker" instead of "Checkboxes"

fixed. @Alexander01998

**Pull request merged**